Adapted for 0.90 from:


From efdec2554b4b65e72de049ea9897ddab5a2b7971 Mon Sep 17 00:00:00 2001
From: Remi Collet <fedora@famillecollet.com>
Date: Tue, 19 Apr 2016 11:30:16 +0200
Subject: [PATCH] Drop Zend\Version usage

This component is no more relevant as it reports its
own version, not the one of the framework.
---
 inc/config.class.php           |   3 +-
 lib/Zend/Version/README.md     |  15 ---
 lib/Zend/Version/Version.php   | 226 -----------------------------------------
 lib/Zend/Version/composer.json |  32 ------
 tools/composer.json            |   1 -
 5 files changed, 1 insertion(+), 276 deletions(-)
 delete mode 100644 lib/Zend/Version/README.md
 delete mode 100644 lib/Zend/Version/Version.php
 delete mode 100644 lib/Zend/Version/composer.json

diff --git a/inc/config.class.php b/inc/config.class.php
index 477ded2..f1008db 100644
--- a/inc/config.class.php
+++ b/inc/config.class.php
@@ -1553,8 +1553,7 @@ static function showLibrariesInformation() {
            "):  ".(class_exists('ezcGraph') ? 'OK' : 'KO'). "\n";
 
       // Zend
-      $zv = new Zend\Version\Version;
-      echo "Zend Framework version " . $zv::VERSION . " in (" . realpath(GLPI_ZEND_PATH) . ")\n";
+      echo "Zend Framework in (" . realpath(GLPI_ZEND_PATH) . ")\n";
 
       // SimplePie :
       $sp = new SimplePie();

